docusaurus.config.js

```diff
@@ -75,6 +75,12 @@ const config = {
                     label: "Talawa API",
                     position: "left",
                 },
+                {
+                  to: '/docs/faq-section',
+                  target: "_self",
+                  label: "FAQ",
```

Comment:
Why was this edited?

Reply:
It is a link in the header for FAQ section, that will redirect users to FAQ item in 'Introduction' below 'About Us' item.